gui/elements.js

```diff
@@ -0,0 +1,65 @@
+const templates = require("./template");
+
+async function showYesNoPopup(description, negative_text, positive_text){
+    // const id = shortUUID.generate().toString();
+    const html = templates.generate_yes_no_popup(description, negative_text, positive_text);
+    const popupDiv = document.getElementById("popup-div");
+    popupDiv.innerHTML = html;
+    const modal = new bootstrap.Modal(document.querySelector("#yes-no-popup"));
```

Comment:
This is making intelliJ unhappy

Reply:
yeah as we are importing it through html and not the node way, can't find a way to fix it other than commenting suppress statement

Reply:
done